modules/gsp-cluster/data/values.yaml

```diff
@@ -152,7 +152,8 @@ prometheus-operator:
           - "alerts-3.monitoring.gds-reliability.engineering"
         scheme: https
   grafana:
-    podAnnotations: { iam.amazonaws.com/role: ${grafana_iam_role_name} }
+    podAnnotations:
+      iam.amazonaws.com/role: ${grafana_iam_role_name}
```

Comment:
According to https://github.com/helm/charts/tree/master/stable/grafana, it says podAnnotations's default value is `{}`. Does this line work without having `{` and `}`? 🤔

Reply:
I think it means the exact same thing, it's just two different syntaxes for a dict/map/object/hash/etc. in YAML